lib/index.js

```diff
@@ -204,6 +204,48 @@ class VideoAsset extends BaseAsset {
 
 exports.VideoAsset = VideoAsset;
 
+class DictionaryAsset extends BaseAsset {
+    constructor() {
+        super();
+        this._object_type = "DictionaryObjectModel";
+        this._content_type = "text/html";
+    }
+
+    set_word (value) { this._word = value; }
+    set_definition (value) { this._definition = value; }
+    set_part_of_speech (value) { this._part_of_speech = value; }
+    set_tags (value) { this._tags = value; }
+
+    to_metadata() {
+        const metadata = super.to_metadata();
+        Object.assign(metadata, {
+            'word': this._word,
+            'definition': this._definition,
```

Comment:
This should be `definition` - see https://github.com/endlessm/endless-content-factory/pull/742/commits/5dc7dcdef80e9e93aecfe5f6fcb7c7d63db4336d

Unless there's something else that needs to change?